crates/validation_common/src/lib.rs

```diff
@@ -0,0 +1,126 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: AGPL-3.0-only
+
+use serde::{Deserialize, Deserializer, Serialize};
+
+#[derive(Clone, Debug, Default, Eq, PartialEq, Serialize)]
```

Comment:
```suggestion
#[derive(Clone, Debug, Default, Eq, PartialEq)]
```

the derived `Serialize` implementation is not valid - this serialization is not symmetric, since `Deserialize` expects a hex-encoded string, whereas this would serialize as a byte blob

Reply:
Removed